src/code.sh

```
@@ -81,150 +53,359 @@ _trim_fastq_endings () {
+
+       duration=$SECONDS
+       echo "All subjobs completed in $(($duration / 60))m$(($duration % 60))s"#
+       IO_PROCESSES=$(nproc --all)
+
+       mark-section "Downloading files from subjobs to the parent job"
+       mkdir subjob_output
+
+       for fusion in "${fusion_lists[@]}"; do
+	       echo $fusion
+	       echo ${fusion%.*}
+	       tar_name=$prefix_$fusion.tar
```

Shellcheck (0.10.0)

[warning] prefix_ is referenced but not assigned (did you mean 'prefix'?).

(SC2154)